Signal.xcodeproj/project.pbxproj

```diff
@@ -1342,6 +1354,16 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		34330A581E7875FB00DF2FB9 /* Fonts */ = {
+			isa = PBXGroup;
+			children = (
+				34330A5B1E787A9800DF2FB9 /* dripicons-v2.ttf */,
+				34330A5D1E787BD800DF2FB9 /* ElegantIcons.ttf */,
+				34330A591E7875FB00DF2FB9 /* fontawesome-webfont.ttf */,
```

Comment:
I added three icon fonts.  We're only using two so far, but I'd be surprised if we didn't use Font Awesome soon too.